src/dialects/oracle/schema/columncompiler.js

```diff
@@ -21,7 +21,8 @@ assign(ColumnCompiler_Oracle.prototype, {
       const tableName = this.tableCompiler.tableNameRaw;
       const createTriggerSQL = Trigger.createAutoIncrementTrigger(
         this.client.logger,
-        tableName
+        tableName,
+        this.client.config.connection.user
```

Comment:
Afaik it might be possible that there is no value in `config.connection.user`.

Reply:
I agree with this statement. We use wallets to connect to Oracle via the `externalAuth: true` parameter in our connection.